daft/ga.py

```diff
@@ -145,3 +147,41 @@ def run(self, **kwargs):
 
         # return the top chromosomes.
         return tools.selBest(population, k=kwargs.pop("k", 1))
+
+
+def combined_mutation(individual, low, up, indpb):
+    """
+    Firstly the type of mutation is choosed with 90% and 10% accordigly.
+    The first mutation type mutates an individual by replacing attributes,
+    with probability *indpb*, by a integer uniformly drawn between *low*
+    and *up* inclusively.The second mutation type mutates a whole individual
+    by replacing all attibuttes by ingegeres uniformly drawn by the initial chain.
+
+    Parameters
+    ----------
+    individual : np.ndarray
+        individual to be mutated.
+    low : int
+        The lower bound or a :term:`python:sequence` of
+        of lower bounds of the range from which to draw the new
+        integer.
+    up : int
+        The upper bound or a :term:`python:sequence` of
+        of upper bounds of the range from which to draw the new
+        integer.
+    indpb : int
+        Independent probability for each attribute to be mutated.
+
+    Returns
+    ----------
+    tuple 
+    """
+
+    # normal mutation with chance of 90%
+    if random.uniform() < 0.9:
```

Comment:
This will not work. `random.uniform()` does not have implicit (default) bounds. It has to be `np.random.uniform()` from NumPy.